extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -596,13 +589,6 @@ export class AmpIframe extends AMP.BaseElement {
     return true;
   }
 
-  /** @override  */
-  viewportCallback(inViewport) {
```

Comment:
according to the offline chat I had with @zhouyx , I think this change is working as intended as amp-iframe does behave differently than amp-ad. @zhouyx pls confirm.

/cc @jridgewell @kristoferbaxter

Reply:
Yes I can confirm that `IntersectionObserverHostApi` is working as expected. 
We are using the native IntersectionObserver within the `IntersectionObserverHostApi` so viewportCallback is no longer needed. 

Also for legacy reason, `IntersectionObserverHostApi` sends update whenever there's an intersection change, but `IntersectionObserverHostForAd` sends update whenever there's a position change when the element is within the viewport. 

we should migrate to using native IntersectionObserver to replace `viewportCallback` for `IntersectionObserverHostForAd` as well.